packages/salesforcedx-utils-vscode/test/unit/auth/authUtil.test.ts

```diff
@@ -7,8 +7,9 @@
 
 import { StateAggregator } from '@salesforce/core';
 import { expect } from 'chai';
-import { createSandbox, SinonSandbox, stub } from 'sinon';
-import { AuthUtil } from '../../../src';
```

Comment:
This is fine for this PR, but in the future don't bother increasing coverage on unit tests and write a jest test for the file instead.